modules/create-new-system-policy.adoc

```diff
@@ -58,7 +58,7 @@ Select *ON* to enforce policy and report a violation. Select *OFF* to only repor
 The enforcement behavior is different for each lifecycle stage.
 
 * For the *Build* stage, {product-title} fails your CI builds when images match the conditions of the policy.
-* For the *Deploy* stage, {product-title} blocks the creation of deployments that match the conditions of the policy. In clusters with admission controller enforcement, the Kubernetes or {ocp} API server blocks all noncompliant deployments. In other clusters, {product-title} edits noncompliant deployments to prevent pods from being scheduled.
+* For the *Deploy* stage, {product-title} blocks the creation of deployments that match the conditions of the policy. In clusters with admission controller enforcement, the Kubernetes or {ocp} API server blocks all noncompliant deployments. In other clusters, {product-title} edits noncompliant deployments to prevent pods from being scheduled. For deployments that already exist, when a violation is found, {product-title-short} does not scale down replicas to enforce the policy.
```

Comment:
The admission controller does block deployment updates on existing deployments where the update violates policies.

Reply:
Can you help me figure out the right wording to use? I will paste the original thread contents below:

The original issue was:

A new policy is introduced in to ACS, which is scoped for certain violation in deployments. For an existing vulnerable deployment, the replicas are not scaled down. However a violation with respect to the policy is still shown in the Console. This seems that the policy does detect the violation but not taking any action.
For a new vulnerable deployment, the policy does comes in to effect and scale down the replicas. And this is also shown in violations window.
Is this an expected behavior? If yes, I hope we have to include this information in the official documentation.

The response was:

This is expected behavior. Policy changes only result in enforcement at the next detection of the criteria.

And once a deployment is running, ACS never scales down. That can only occur at the time where the Deployment is created to prevent the Pods from starting.

Reply:
Rewritten after talking with Khushboo.